### PR TITLE
make tests compatible with having correct soma

### PR DESCRIPTION
**Pull request by mgeplf**
* many "contour" soma had only a single point

### CODE REVIEW

File: tests/test_graft.py [hunk 1]
```diff
@@ -95,8 +95,9 @@ def test_graft_axon_on_synthesized_cell():
     graft.graft_axon(synthesized_cell, donor_neuron)
     axon = graft.find_axon(synthesized_cell)
     assert_array_almost_equal(axon.points,
-                              [[5.1272364, 5.4825425, 4.9689593],
-                               [5.1272364, 1.4825425, 4.9689593]])
+                              [[1.669128, 8.830159, 4.592799],
```
Comment (mgeplf) [inline tests/test_graft.py:98]: this is the only change that I'm uncomfortable with; I'm not sure why the expectation was correct, or whether this is correct.

File: tests/data/input-convert/simple2.asc [hunk 2]
```diff
@@ -1,7 +1,10 @@
 ("CellBody"
  (Color Red)
  (CellBody)
- (0 0 0 2)
+ ( 0.1  0   0 2)
+ ( 0.1  0.1 0 2)
+ (-0.1  0   0 2)
+ (-0.1 -0.1 0 2)
```
Comment (adrien-berchet) [inline tests/data/input-convert/simple2.asc:7]: Isn't the shape of this soma weird?
Why not something like this?
```suggestion
 ( 0.1  0   0 2)
 ( 0  0.1 0 2)
 (-0.1  0   0 2)
 (0 -0.1 0 2)
```

Comment (mgeplf) [reply tests/data/input-convert/simple2.asc:7]: yeah, good  eye; it is weird - interestingly / unsurprisingly the tests didn't mind...
I'll prob. switch to something more sane

Comment (mgeplf) [reply tests/data/input-convert/simple2.asc]: done

File: tests/test_graft.py (hunk 1 above)
Comment (mgeplf) [reply tests/test_graft.py:98]: @eleftherioszisis; can you have a look at this?

Comment (eleftherioszisis) [reply tests/test_graft.py]: As discussed this was expected because the soma radius was different.

**Review by eleftherioszisis — APPROVED** [review]

**Pull request merged**